src/test/kotlin/no/nav/tilleggsstonader/sak/behandlingsflyt/BehandlingFlytTest.kt

```diff
@@ -320,6 +321,12 @@ class BehandlingFlytTest : IntegrationTest() {
                 tom = LocalDate.of(2024, 1, 31),
                 utgift = 1000,
                 erFremtidigUtgift = false,
+                offentligTransport =
+                    OffentligTransport(
+                        reisedagerPerUke = 2,
+                        prisEnkelbillett = 44,
+                        prisTrettidagersbillett = 750,
+                    ),
```

Comment:
Tror det kan bli litt forvirrende med offentlig transport på et pass barn vilkår. Foreslår å bruke `null` her også heller teste offentlig transport vilkåret et annet sted
```suggestion
                offentligTransport = null,
```